Makefile

```diff
@@ -185,7 +185,7 @@ test-unit: install-deps
 test-integration: install-deps build-python-packages	
 	cd $(TESTS_DIR)
 	$(POETRY) run $(PYTHON) -m env_control create
-	$(POETRY) run behave --show-timings --stop -D skip_setup --junit $(BEHAVE_ARGS)
```

Comment:
Do we still need to execute `env_control create` after removing `-D skip_setup`?